modules/apps/29-fee/client/cli/query.go

```diff
@@ -189,3 +189,78 @@ func GetCmdCounterpartyAddress() *cobra.Command {
 
 	return cmd
 }
+
+// GetCmdFeeEnabledChannels returns the command handler for the Query/FeeEnabledChannels rpc.
+func GetCmdFeeEnabledChannels() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "channels",
+		Short:   "Query the ibc-fee enabled channels",
+		Long:    "Query the ibc-fee enabled channels",
+		Args:    cobra.NoArgs,
+		Example: fmt.Sprintf("%s query ibc-fee channels", version.AppName),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientQueryContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			pageReq, err := client.ReadPageRequest(cmd.Flags())
+			if err != nil {
```

Comment:
I think if we return an err here this means the pagination flags are not optional. Is that what we want?